Vagrantfile

```diff
@@ -148,15 +139,21 @@ Vagrant.configure("2") do |config|
     run: "always"
 
   config.vm.provision "shell",
-    inline: "kill $(pgrep 'unattended'); apt-get update; apt-get install -y mysql-client"
-  config.env.enable
-  config.vm.provision "docker" do |d|
-    d.post_install_provision "shell",
-      inline:"docker network create devopsloft_network"
-  end
+    inline: "apt-get update; apt-get install -y mysql-client"
 
-  config.vm.provision "shell",
-    inline: $dcompose, run: "always"
+  config.env.enable
+  config.vm.provision :docker
+  config.vm.provision :docker_compose,
+    compose_version: "1.24.0"
+  config.vm.provision "app",
+    type: "shell",
```

Comment:
why are you using shell instead of 
`  config.vm.provision :docker_compose, yml: "/vagrant/docker-compose.yml", rebuild: true, run: "always"`

Reply:
because docker-compose consumes the .env file only when it is executed in the same folder.
The vagrant-docker-compose plugin execute it from the home directory instead of in /vagrant.

Reply:
https://github.com/leighmcculloch/vagrant-docker-compose/issues/52

Reply:
so, maybe we can just copy the .env file before ?

Reply:
provide a PR with the suggested change